encoder/vaapiencoder_vp9.cpp

```diff
@@ -159,9 +157,7 @@ YamiStatus VaapiEncoderVP9::doEncode(const SurfacePtr& surface,
 
     PicturePtr picture(new VaapiEncPictureVP9(m_context, surface, timeStamp));
 
-    m_frameCount %= keyFramePeriod();
-    picture->m_type = (m_frameCount ? VAAPI_PICTURE_P : VAAPI_PICTURE_I);
-    m_frameCount++;
+    picture->m_type = (forceKeyFrame ? VAAPI_PICTURE_I : VAAPI_PICTURE_P);
```

Comment:
Force Key frame usually use by realtime communication program. If there are some frame drop between peers, following frames are not decodeable, until peer see a key frame. So If peer detect a frame dropping, It will sent a notification back to yami’s user,  then the user will sent an forceKeyFrame to yami. Therefore, we’d better have two machnism here.
1.	The normal case, the I frame decide by intraPeriod
2.	If user want force key frame, we will start a new gop.
Please refer to https://github.com/01org/libyami/blob/apache/encoder/vaapiencoder_h264.cpp#L1154
And https://github.com/01org/libyami/blob/apache/encoder/vaapiencoder_h264.cpp#L1279